.changesets/add-noop-mode-for-unsupported-systems.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "add"
+---
+
+Unsupported systems as Windows won't start the agent and other integration components to prevent them from failing and allowing apps to be run normally.
```

Comment:
```suggestion
Unsupported systems, like Microsoft Windows, won't start the agent and other integration components to prevent them from failing and allowing apps to be run normally.
```